bindings/python/malamute/_malamute_ctypes.py

```diff
@@ -7,7 +7,7 @@
 import os, sys
 from ctypes import *
 from ctypes.util import find_library
-import czmq
+import czmq.42ity
```

Comment:
it looks like you have a czmq.42ity directory as well - I have the same problem, you'll need to call it 42ity-czmq instead - for some reasons the system picks czmq* before czmq